packages/amplify-codegen-appsync-model-plugin/src/preset.ts

```diff
@@ -2,9 +2,10 @@ import { Types } from '@graphql-codegen/plugin-helpers';
 import { Kind, TypeDefinitionNode } from 'graphql';
 import { join } from 'path';
 import { JAVA_SCALAR_MAP, SWIFT_SCALAR_MAP, TYPESCRIPT_SCALAR_MAP } from './scalars';
+import { LOADER_CLASS_NAME } from './configs/java-config';
 
 const APPSYNC_DATA_STORE_CODEGEN_TARGETS = ['java', 'android', 'swift', 'ios', 'javascript', 'typescript'];
-const JAVA_PACKAGE_NAME = 'com.amplify.datastore.generated';
+const JAVA_PACKAGE_NAME = 'com.amplifyframework.datastore.generated.model';
```

Comment:
can we define it once, then reference? reused string literals should be shared (applies to others as well)